src/rent-vs-mortgage.md

```diff
@@ -362,44 +356,62 @@ Inputs.table(search, {
 </div>
 
 
-## Where are the mortages "affordable" (closest to rents)?
+## Where are the mortages relative "affordable" (closest to rents)?
 
-<div class="card" style="margin: 0 -1rem;">
+```js
+const propertyType = view(Inputs.radio(['HOUSE', 'UNIT'], { label: 'Property Type', value: 'HOUSE' }))
+const rangeType = view(Inputs.radio(['DYNAMIC', 'FIXED'], { label: 'Colour Range Type', value: 'FIXED' }))
+```
+
+The FIXED colour range option ensures the same fixed range of colours for consistent display across both units and houses, while the DYNAMIC option adjusts the range based on the data to maximise the contrast.
+
+```js
+const dataArray = data.toArray().filter(row => row.property_type === propertyType)
+const sa2Map = new Map(dataArray.map(row => [row.sa2_code.toString(), row]))
+const sa2Codes = new Set(sa2Map.keys())
+const domain = rangeType === 'FIXED'
+   ? FIXED_DOMAIN
+   : dataArray.reduce((acc, row) => {
+        const ratio = row.repayment_to_rent_ratio
+        return [Math.min(acc[0], ratio), Math.max(acc[1], ratio)]
+      },
+      [100, 0],
+   )
```

Comment:
shouldn't Plot dynamically compute the domain for the color scale here? We could yield `undefined` on the else condition if that's the case

Reply:
Oh, that's a great point - definitely easier to leave it to Plot in that case, I'll check it out.

Reply:
Yeah that works, just need to actually plug in some data into the unrendered plot below that I use to get the range.